warehouse/query-core/src/main/java/datawave/query/jexl/DatawaveInterpreter.java

```diff
@@ -254,52 +261,68 @@ public Object visit(ASTOrNode node, Object data) {
         return result;
     }
     
-    public Object interpretOr(Object left, Object right) {
-        FunctionalSet leftFunctionalSet = null;
-        FunctionalSet rightFunctionalSet = null;
-        if (left == null)
-            left = FunctionalSet.empty();
-        if (!(left instanceof Collection)) {
-            try {
-                boolean leftValue = arithmetic.toBoolean(left);
-                if (leftValue) {
-                    return Boolean.TRUE;
-                }
-            } catch (ArithmeticException xrt) {
-                throw new RuntimeException(left.toString() + " boolean coercion error", xrt);
-            }
-        } else {
-            leftFunctionalSet = new FunctionalSet();
-            leftFunctionalSet.addAll((Collection) left);
-        }
-        if (right == null)
-            right = FunctionalSet.empty();
-        if (!(right instanceof Collection)) {
-            try {
-                boolean rightValue = arithmetic.toBoolean(right);
-                if (rightValue) {
-                    return Boolean.TRUE;
-                }
-            } catch (ArithmeticException xrt) {
-                throw new RuntimeException(right.toString() + " boolean coercion error", xrt);
-            }
-        } else {
-            rightFunctionalSet = new FunctionalSet();
-            rightFunctionalSet.addAll((Collection) right);
-        }
+    /**
+     * Determine the result of or'ing left with right. null means no result, empty functional set is essentially a false, otherwise true.
+     * 
+     * @param left
```

Comment:
to piggyback off the annotations comment, could the parameters be formatted to have a "description" line for each so the mvn warnings don't get tripped? something like:

     * @param left
     *                    left param
     * @param right
     *                   right param